src/main/java/games/strategy/util/PropertyUtil.java

```diff
@@ -130,15 +130,14 @@ private static Method getSetter(final String propertyName, final Object subject,
 
   private static Method getResetter(final String propertyName, final Object subject) {
     final String resetterName = "reset" + capitalizeFirstLetter(propertyName);
-    // for (final Method c : subject.getClass().getDeclaredMethods())
     for (final Method c : subject.getClass().getMethods()) {
       if (c.getName().equals(resetterName)) {
         try {
           return subject.getClass().getMethod(resetterName);
-        } catch (final NoSuchMethodException nsmf) {
-          // Go ahead and try the first one
-          return c;
-        } catch (final NullPointerException n) {
+        } catch (final NoSuchMethodException | NullPointerException nsmf) {
```

Comment:
Sorry, thought I tagged this line in my previous review...  Same as the change in 6383cda.